Readme.md

```diff
@@ -17,10 +17,10 @@ This branch of RxSwiftExt targets Swift 4.x and RxSwift 4.0.0 or later.
 
 #### CocoaPods
 
-Using Swift 4:
+Add to your `Podfile`:
 
 ```ruby
-pod 'RxSwiftExt'
+pod 'RxSwiftExt', '~> 5'
```

Comment:
I think this should be '~> 4' (Swift v5, RxSwiftEx v4)

Reply:
Yup. This was originally when I thought we'll actually bump staright to RxSwift 5, that's what I thought of doing as well 🤔